src/main/java/org/sonar/plugins/ldap/windows/WindowsAuthenticationHelper.java

```diff
@@ -235,7 +235,7 @@ UserDetails getUserDetails(IWindowsAccount windowsAccount) {
 
     Map<String, String> adUserDetails = getAdUserDetails(windowsAccount.getDomain(), windowsAccount.getName());
     if (!adUserDetails.isEmpty()) {
-      userDetails.setName(adUserDetails.get(AdConnectionHelper.COMMON_NAME_ATTRIBUTE));
+      userDetails.setName(adUserDetails.get(AdConnectionHelper.REAL_NAME_ATTRIBUTE));
```

Comment:
It would be good if we can take REAL_NAME_ATTRIBUTE from WindowsAuthSettings. The reason for this is there will be some users who will like to get Real name from "cn" . We can have one method getLdapUserRealNameAttribute() in WindowsAuthSettings  which can return default value of REAL_NAME_ATTRIBUTE

/**
   * Setting to specify the real name attribute, which will be used to fetch the user's real name
   */
  public static final String LDAP_WINDOWS_USER_REAL_NAME_ATTRIBUTE = LDAP_WINDOWS + ".user.realNameAttribute";
public static final String DEFAULT_USER_REAL_NAME_ATTRIBUTE = "cn";

/**
   * Setting to specify the user's real-name attribute, By default, its value is "cn".
   */
  public String getLdapUserRealNameAttribute(){
    return StringUtils.defaultIfBlank(settings.getString(LDAP_WINDOWS_USER_REAL_NAME_ATTRIBUTE), DEFAULT_USER_REAL_NAME_ATTRIBUTE );
  }